### PR TITLE
Makes seclite attachable to operator helmet

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

What it says on the tin.
Attached by clicking with flashlight on the helmet.
Removed by dragging the helmet into empty hand.
Toggled by Alt-clicking the helmet.

image

https://user-images.githubusercontent.com/65828539/148748442-5407cc37-7d6d-4f1f-b726-4306c4987b5f.mp4

## Why It's Good For The Game

I honestly don't know.

## Changelog
:cl:
add: ability to attach seclite to operator helmet
/:cl:

### CODE REVIEW

File: code/modules/clothing/head/armor.dm [hunk 1]
```diff
@@ -75,6 +75,56 @@
...
+			on = TRUE
+			ping_flashlight()
+		else
+			on = FALSE
+		update_icon()
+		return
+	..()
+
+/obj/item/clothing/head/armor/helmet/ironhammer/attackby(obj/item/I, mob/user)
+	if(istype(I, fitting_flashlight))
```
Comment (SirRichardFrancis) [inline code/modules/clothing/head/armor.dm:104]:
```suggestion
	if(istype(I, fitting_flashlight) && !flashlight)
```

**Comment by CeUvi** [comment]
now make a "toggle helmet light" action button

**Comment by SirRichardFrancis** [comment]
> now make a "toggle helmet light" action button

Make a light overlay for that helmet and we have a deal.

**Comment by TheShown911** [comment]
Just put it in your pocket.

**Review by Firefox13 — CHANGES_REQUESTED** [review]
Needs a (preferably) sprite and flavour desc (There is a flashlight attached to the helmet, alt+click (or whatever it is) to remove).
Adding `description_info` to both helmet and seclites that it can have a flashlight attached/it can be attached to helmet would be awesome too

**Pull request closed without merging**